scapy/contrib/automotive/bmw/hsfz.py

```diff
@@ -10,7 +10,10 @@
 import struct
 import socket
 import time
-from scapy.packet import Packet, bind_layers, bind_bottom_up
+
+from typing import Tuple, Any
```

Comment:
The `typing` module isn't available by default on Python 2.
But it seems the cryptography module (or one of its dependencies) installs it, making those tests pass.
Right now, we have the `scapy.compat` trickeries. I wonder if we shouldn't drop those and just add `typing` as a mandatory python 2 dependency.

Reply:
Using `typing` instead of `scapy.compat` would result in better support of my IDE (pycharm). But I can also change that. What should I do?

Reply:
>  would result in better support of my IDE (pycharm)
Interesting. How different is it ?

I checked it again and there's actually real issues as to why we're using `scapy.compat` right now, see
https://github.com/secdev/scapy/blob/b458ebc01d31471ad15dfb2230653c7347ec0306/scapy/compat.py#L78-L81

You should replace the `from typing import X` by `from scapy.compat import X`

Reply:
alright. I will change that. Thanks

Reply:
Done.